internal/params/config.go

```diff
@@ -638,3 +643,12 @@ func (c *ChainConfig) Rules(epoch *big.Int) Rules {
 		IsSHA3:       c.IsSHA3(epoch),
 	}
 }
+
+func (c *ChainConfig) EnablePruneBeaconChainFeature() {
+	c.pruneBeaconChainEnable = true
```

Comment:
This shall not be part of chainConfig. Keep it within `HarmonyConfig`. `ChainConfig` is all about chain fork params, pruning is not effecting chain config IMO.

Reply:
Can you reorg it based on Jacky's comment?

Reply:
fixed